doc/source/getting_started/install.rst

```diff
@@ -140,7 +140,7 @@ Installing with ActivePython
 Installation instructions for
 `ActivePython <https://www.activestate.com/activepython>`__ can be found
 `here <https://www.activestate.com/activepython/downloads>`__. Versions
-2.7 and 3.5 include pandas.
+2.7, 3.5 and 3.6 include pandas.
```

Comment:
is this correct?

Reply:
In https://www.activestate.com/products/python/ they say they include pandas, and they now have a 3.6 version, so it should be true.